docs/release/2022q2.md

```diff
@@ -0,0 +1,155 @@
+## 2022-07-01: 2022q2 release
+
+This release covers everything from 2022-04-01 and has been tested with avr-gcc
+v11.2.0 from upstream and arm-none-eabi-gcc 10.3-2021.10 from Arm.
+
+Breaking changes:
+
+- None.
+
+Features:
+
+- Custom OpenOCD binary path.
+- IRQ handlers for RP2040.
+- Expose transmission complete flag on STM32 UART.
+- Half transfer handler for STM32 DMA.
+- Timer `SlaveModeTrigger::Internal0`/`Internal3` for STM32F4.
+- UI Button group refactoring.
+- MovingAverage filter refactoring.
+- Support for std::chrono in STM32 timer API.
+
+Integrated Projects:
+
+- ETL upgraded to v20.29.3.
+- CrashCatcher updated to latest version.
+
+Fixes:
+
+- STM32H7 RCC clock output.
+- CAN system clock frequencies for Nucleo-F446RE/Nucleo-F446ZE boards.
+- Cortex-M ITM when `buffer.tx` option not set.
+
+New development boards:
+
+- Support for revision B-03 of the DISCO-F469NI development board as
+  `modm:disco-f469ni:b-02`.
+
+New device drivers:
+
+- MAX31855 thermocouple-to-digital converter
```

Comment:
```suggestion
- MAX31855 thermocouple-to-digital converter as `modm:driver:max31855`
```
or whatever the module name is, then it gets linked automatically in the changelog.
etc for all